packages/homepage/src/pages/blog/_elements.js

```diff
@@ -1,99 +1,124 @@
-
-export const Posts = styled.article`
-  display: flex;
-  align-items: flex-start;
-  background: ${props => props.theme.background2};
-  border-radius: 4px;
-  padding: 1rem;
-  margin-bottom: 30px;
-
-  @media screen and (max-width: 1200px) {
-    flex-direction: column;
+import styled from 'styled-components';
+
+export const Header = styled.section`
+  text-align: center;
+  padding: 1rem 0rem;
+  color: rgb(242, 242, 242);
+  margin-bottom: 2rem;
+  word-break: normal;
+
+  @media screen and (min-width: 768px) {
+    padding: 4rem 0rem;
+  }
+
+  @media screen and (min-width: 960px) {
+    padding: 8rem;
   }
 `;
 
-export const Subtitle = styled.h3`
-  font-weight: 500;
-  font-size: 1rem;
-  line-height: 1.5;
+export const PageTitle = styled.h1`
+  font-size: 2.5rem;
+  line-height: 3rem;
+  font-weight: 700;
 
-  color: #b8b9ba;
-`;
+  letter-spacing: -0.02rem;
 
-export const Thumbnail = styled.img`
-  margin-right: 25px;
+  @media screen and (min-width: 768px) {
+    font-size: 4rem;
+    line-height: 5rem;
+  }
 
-  @media screen and (max-width: 1200px) {
-    margin-bottom: 25px;
-    width: 100%;
+  @media screen and (min-width: 960px) {
+    font-size: 4.5rem;
+    line-height: 5.5rem;
   }
 `;
 
-export const Aside = styled.aside`
-  section {
-    display: flex;
-    align-items: center;
-  }
+export const PageSubtitle = styled.h3`
+  font-size: 1.5rem;
+  line-height: 1.5;
+  font-weight: 400;
+  padding: 0 2rem;
+  margin: 0 0 1.0875rem;
+}
 
-  ${({ mobile }) =>
-    !mobile &&
-    css`
-      @media screen and (max-width: 850px) {
-        display: none;
-      }
-    `};
-  ${({ mobile }) =>
-    mobile &&
-    css`
-      display: flex;
-      align-items: flex-start;
-      justify-content: space-between;
-      flex-direction: row-reverse;
-      margin-top: 25px;
-      width: 100%;
-      @media screen and (min-width: 851px) {
-        display: none;
-      }
-    `};
+@media screen and (min-width: 768px) {
+  font-size: 2rem;
+  line-height: 1.5;
+  }
 `;
 
-export const Wrapper = styled.div`
+export const Grid = styled.div`
+  max-width: 960px;
+  margin: 0 auto;
+  width: 100%;
   display: grid;
-  grid-template-columns: 250px 1fr;
-  grid-gap: 90px;
+  grid-template-columns: 1fr;
+  grid-column-gap: 2rem;
+  grid-row-gap: 4rem;
 
-  @media screen and (max-width: 850px) {
-    grid-template-columns: 1fr;
+  > div {
+    transform: scale(1);
+    transition: all 0.2s ease-in-out;
   }
-`;
 
-export const Header = styled.section`
-  background-color: #0971f1;
-  border-radius: 4px;
-  padding: 30px;
-  color: #f2f2f2;
-
-  margin-bottom: 30px;
+  > div :hover {
+    transform: scale(0.98);
+    box-shadow: 0 0.125rem 0.5rem rgba(0, 0, 0, 0.25);
+  }
 
-  width: calc(100% - 340px);
-  margin-left: 340px;
+  @media screen and (min-width: 768px) {
+    grid-template-columns: 1fr 1fr;
 
-  @media screen and (max-width: 850px) {
-    width: 100%;
-    margin-left: 0;
+    > div:nth-child(-n + 1) {
```

Comment:
Very nice!!!

Reply:
oh, you can also use `:first-child` or didn't it work?

Reply:
Ah yes i first tried to do the latest 3 bigger, but decided to only do the fist one. I did not change because I might want to change later 😄